loggingpy/sink.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import requests
+
+
+class HttpSink(logging.Handler):
+
+    def __init__(self, endpoint_uri: str):
+        logging.Handler.__init__(self)
+        self.endpoint_uri = endpoint_uri
+
+    def emit(self, record):
+        log_entry = self.format(record)
+
+        return requests.post(self.endpoint_uri, log_entry, headers={"Content-type": "application/json"}).content
```

Comment:
Does requests have some resiliency features built-in? If not, we should use / build some helper methods that can do some retries if the initial call fails (for all our API calls).

Reply:
They have some basic resiliency due to retries, but this can be configured to improve it. Per default, it retries up to a certain time out where it stops retrying. Here are some possibilities of what it can do by configuration: https://www.peterbe.com/plog/best-practice-with-retries-with-requests

Reply:
some basic retries should be good enough for now I think. we can still take our learnings. for example, our whole logging gets botched if the service gets offline, but i'm not sure how much of an issue this is in practice (if the service is *that* dead, the reason is probably not in the logs).

Reply:
Another issue to me is definitely that if the requests can no longer be sent to the logging provider, the queue in my logger gets filled to the brim (of the memory in fact). But I think we do not want a log message drop strategy, right?

Reply:
We're actually dropping them in .NET if delivery fails after a number of retries. I think a prime directive is that logging *must never* bring down a production app. A more sophisticated algorithm would be to just persist them on file or smth, but that'll tank anyway depending on our hosting environments.

Reply:
Should I already implement such a dropping strategy?